### DISCUSSION TITLE
Looking forward to testing out the new version!

### DISCUSSION THREAD

**Discussion by ghost** [geckosio/geckos.io#127] [General]
Just saw that there was an overhaul. Can't wait to try it out! :)

**Comment by ghost** [comment]
Getting there but also taking this time to rewrite my game. Also finally got it running on DigitalOcean droplet too! :)

```
import express from 'express';
import path from 'path';
import http from 'http';
import geckos from '@geckos.io/server';

const port = 5000;
const __dirname = path.resolve();

// create express app
const app = express();
app.use(express.static(path.join(__dirname, 'public')));
app.get('/', (req, res) => {
    res.sendFile(`${__dirname}/public/index.html`);
});
const server = http.createServer(app);

const io = geckos();
io.addServer(server);

io.onConnection(channel => {
  console.log(`Client ${channel.id} connected.`);
});

server.listen(port, () => {
  console.log(`Application listening on port ${port}.`);
});
```

**Reply by yandeu** [reply]
I'm using `__dirname` and `__filename` as following:

```ts
import { dirname } from 'path'
import { fileURLToPath } from 'url'
const __filename = fileURLToPath(import.meta.url)
const __dirname = dirname(__filename)
```

**Comment by htmn** [comment]
It would be nice if you let us know how many players you're able to serve simultaneously, what droplet are you using and what was CPU usage :)

**Comment by ghost** [comment]
Hi htmn. I haven't done any player load testing yet. Still in the early stages even though it's quite playable. Players can join, fly around, and shoot each other. I'm using DigitalOcean droplet (Premium AMD with NVMe SSD, 1GB RAM, 25 GB SSD, 1000GB transfer @ $6/month.
